datacapture/src/main/java/com/google/android/fhir/datacapture/fhirpath/ExpressionEvaluator.kt

```diff
@@ -256,6 +252,69 @@ object ExpressionEvaluator {
     return evaluateVariable(expression, questionnaireResponse, variablesMap)
   }
 
+  /**
+   * Create an x-fhir-query string for evaluation
```

Comment:
```suggestion
   * Creates an x-fhir-query string for evaluation.
```

and apply for the rest of the PR